src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml

```diff
@@ -206,7 +206,7 @@
 											  Margin="0,0,6,6"
 											  Background="Transparent"
 											  CornerRadius="{TemplateBinding CornerRadius}"
-											  Elevation="6">
+											  Elevation="0">
```

Comment:
This is just hiding the problem, we will still have a white border around anything that is elevated. And the FAB should have an elevation set in order to show the shadow, no?

Reply:
Does this means that white border is elevated to everything  not only FAB buttons.